app/models/filters/hud_filter_base.rb

```diff
@@ -21,6 +22,13 @@ def effective_project_ids
       @effective_project_ids += effective_project_ids_from_data_sources
       @effective_project_ids += effective_project_ids_from_project_types
 
+      if funder_ids.present?
+        @effective_project_ids = funder_scope.where(id: funder_ids).
+          joins(:project).
+          where(p_t[:id].in(@effective_project_ids)).
+          pluck(p_t[:id])
+      end
```

Comment:
Is there a more efficient way to do this?

Reply:
That looks good to me.  Maybe some slight tweaks
```suggestion
      # limit to projects funded by the selected projects
      if funder_ids.present?
        @effective_project_ids = funder_scope.where(id: funder_ids).
          joins(:project).
          where(p_t[:id].in(@effective_project_ids)).
          distinct.
          pluck(p_t[:id])
      end
```

Alternatively, you could flip it around, and 
```
@effective_project_ids =  GrdaWarehouse::Hud::Project.joins(:funder).
  where(id: @effective_project_ids).
  where(f_t[:id].in(funder_ids).
  distinct.
  pluck(:id)
```
but the only benefit to that is that it's more obvious you are pulling project ids since it's the main active record model.

Reply:
Alright, interesting! Updated with the suggestion to use distinct.